### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by fatmabadri**

### CODE REVIEW

**Comment by fatmabadri** [comment]
Hi @alioguzhan,
Would you make me enable to assign someone and add label to the PR?

**Comment by alioguzhan** [comment]
Hi @fatmabadri 

You don't need to make assignments. Since we are only 2 (@yunusemredilber and I), we can manage that process internally. Thanks for your interest.

**Comment by yunusemredilber** [comment]
I have one more suggestion. I think there are a lot more commits than necessary for this translation. To keep the commit history clear, It might make sense to do "Squash and Merge" at the end. What do you think @alioguzhan ?

**Comment by alioguzhan** [comment]
Yes. That makes sense. Let's squash and merge once this is ready to merge @yunusemredilber  

Since you are translating one document, it is better if you keep it under one commit or two (with later updates). @fatmabadri

**Comment by fatmabadri** [comment]
> 
> 
> Hi @fatmabadri
> 
> You don't need to make assignments. Since we are only 2 (@yunusemredilber and I), we can manage that process internally. Thanks for your interest.

Hi @alioguzhan. I have asked to be enabled to add the reviewer myself when making a PR not to make an assignment or something else. This is my last translation. I am not interested to join you internally since you are only two people, managed to the end and the Turkish translation is almost done. Thank you.

**Comment by alioguzhan** [comment]
@fatmabadri I thought you were mentioning picking a user from maintainers to review this PR. I didn't think that you would want to assign yourself as a reviewer to your own Pull Request. 

Someone else has to be a reviewer for our own PR's. A second or third eye re-views the work that has been done and checks if this is appropriate to be merged. This is the idea about Pull Request and Review process.

Thanks for your work and translation. I will review it as soon as I have some spare time.

**Comment by fatmabadri** [comment]
> @fatmabadri I thought you were mentioning picking a user from maintainers to review this PR. I didn't think that you would want to assign yourself as a reviewer to your own Pull Request.
> 
> Someone else has to be a reviewer for our own PR's. A second or third eye re-views the work that has been done and checks if this is appropriate to be merged. This is the idea about Pull Request and Review process.
> 
> Thanks for your work and translation. I will review it as soon as I have some spare time.

@alioguzhan, 
You have misunderstood. I asked to be enabled to add the reviewer myself, I did not say to add myself as a reviewer. I intended to add the maintainer while a PR was being made. 
I cannot be the reviewer of my own translation. There is no point to do that. Absolutely, my translation has to be reviewed by the maintainer as they are the ones responsible and have more experience to do this. I hope it is clear now. And also thank you for your time.

**Comment by fatmabadri** [comment]
> Yes. That makes sense. Let's squash and merge once this is ready to merge @yunusemredilber
> 
> Since you are translating one document, it is better if you keep it under one commit or two (with later updates). @fatmabadri

It is one document but it is a quite long one. Translations were made in the different days. Multiple commits were done in order to keep the translated part protected . @alioguzhan

**Comment by alioguzhan** [comment]
> > Yes. That makes sense. Let's squash and merge once this is ready to merge @yunusemredilber
> > Since you are translating one document, it is better if you keep it under one commit or two (with later updates). @fatmabadri
> 
> It is one document but it is a quite long one. Translations were made in the different days. Multiple commits were done in order to keep the translated part protected . @alioguzhan

It is totally fine if this is your preferred way. As I said, we can squash and merge. No problem at all. 

> > @alioguzhan,
> > You have misunderstood. I asked to be enabled to add the reviewer myself, I did not say to add myself as a reviewer. I intended to add the maintainer while a PR was being made.

I see. So this brings us to my previous answer. Since we do not know who is available or not, It becomes uncertain who will be reviewing some particular PR. If the contributors would have permission to add reviewers, it would be a `demand` more than an `assignment`. And this is a concept we don't have here. In our case, I or @yunusemredilber are assigning ourselves. Or we mention each other in a comment by asking kindly if he can review this or not. 

Sorry for the misunderstanding.

File: content/docs/accessibility.md
Comment (alioguzhan) [inline content/docs/accessibility.md:137]: what about `konusuna deginir` or `bahseder` ?

File: content/docs/accessibility.md
Comment (alioguzhan) [inline content/docs/accessibility.md:161]: `Odagi programlayarak yonetmek` sounds more accurate. What do you think ?

File: content/docs/accessibility.md
Comment (alioguzhan) [inline content/docs/accessibility.md:246]: typo in `genelliklre`

File: content/docs/accessibility.md
Comment (alioguzhan) [inline content/docs/accessibility.md:322]: space after `(popover)`

File: content/docs/accessibility.md
Comment (alioguzhan) [inline content/docs/accessibility.md:363]: This should be better:

`Bu kod, işlevselliği hem imlece hem de klavye kullanıcılarına sunar.`

File: content/docs/accessibility.md
Comment (alioguzhan) [inline content/docs/accessibility.md:365]: space after `(popover)`

File: content/docs/accessibility.md
Comment (alioguzhan) [inline content/docs/accessibility.md:382]: This is a very nice translation. Thanks.

File: content/docs/accessibility.md
Comment (alioguzhan) [inline content/docs/accessibility.md:384]: typo in `ayarlanmak`

File: content/docs/accessibility.md
Comment (alioguzhan) [inline content/docs/accessibility.md:423]: `Farenizin`

**Review by alioguzhan — CHANGES_REQUESTED** [review]
Thanks for the translation. See my comments, please.

File: content/docs/accessibility.md
Comment (fatmabadri) [reply content/docs/accessibility.md:137]: verb of 'deginir' is like to touch something lightly
verb of 'bahseder' sounds like giving a brief explanation.
verb of 'konusur' is the exact Turkish translation to 'talks' within the title.
If  you are not satisfied with the exact translation, you are free to replace it with the choice of yours.

File: content/docs/accessibility.md
Comment (fatmabadri) [reply content/docs/accessibility.md:363]: This was my first choice, what I was told that not to use word of 'işlev'

File: content/docs/accessibility.md
Comment (fatmabadri) [reply content/docs/accessibility.md:161]: the primary point of the title is the 'managing focus'. So, keeping the words of the primary part without divided, my suggestion would be: 'programlayarak odağı yönetmek'. I will apply this, if you still don't feel  that it sounds accurate, it can be changed to your choice.

File: content/docs/accessibility.md
Comment (alioguzhan) [reply content/docs/accessibility.md]: You can use `fonksiyonellik` or `islev`. That was not the problem in that sentence. 

I wanted to change the way you translate the word `expose`. Also, the first translation meant a totally different thing if you look at the original sentence.

Comment (fatmabadri) [reply content/docs/accessibility.md]: At the first look, I have seen the replaced word of `islevsellgini`. If it was not a problem to use either one, why did you change it? I thought, this was the change you were asking for. Your suggestion could be descriptive or to be targeted on the word what you wanted to change.
What might be the English word to your suggested word of `sunar`? `Provide`?.. 
`Provide` and `expose` have completely different meanings.
What is the meaning of `expose`? [Cambridge dictionary](https://dictionary.cambridge.org/dictionary/english-turkish/expose) and [Oxford dictionary](https://www.oxfordlearnersdictionaries.com/definition/english/expose_1?q=expose):`to show something hidden`
My sentence is grammatically wrong that gives different meaning because of wrong suffixes. It is not the way I translate the word of `expose`, it is the the way I translate the objects of  `pointer device` and `keyboard users`.
The way you translate the word of `expose` excludes the context of the document and the meaning of the original word by adding another meaning to make the sentence sounds well. 
I do not agree with your translation however, there is no need to keep this conversation going. I have to respect you time, changes you are asking will be applied. Thank you.

**Comment by alioguzhan** [comment]
Thanks for the translation. (Forgot to write this message earlier today)

**Pull request merged**
